### DISCUSSION TITLE
Allow autoMinMax for percent percentunit types

### DISCUSSION THREAD

**Discussion by knoguchi** [grafana/grafana#38374] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
Remove the full scale min and max for the percent and percentunit units.

**Why is this needed**:
BarGauge panel is more readable when the range is not fixed for min=0 and max=100%.  For example when the values are around 1.0% the bar gauge is unreadable.

Same problem for the spark line of the Stat panel.   I have a stat panel for a metric where the value is usually 1%, and small deviation is significant. The spark line is kind of flat because the value is low.  I can't simply set min and max because it can shoot up to 90% occasionally. 

https://github.com/grafana/grafana/blob/master/packages/grafana-data/src/field/fieldOverrides.ts#L160-L175

The above section forces the min=0 and max=100 for precent or max=1 for percentunit when min and/or max is not set.   I think it's not necessary.  Users can enter 0 and 100 if they want this behavior.

**Comment by marefr** [comment]
@peterholmberg why have this been added to the backend platform backlog?

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
